core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -47,12 +47,25 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
       <a ng-repeat="node in $ctrl.collection.getCollectionNodes()"
          ng-href="<[$ctrl.getExplorationUrl(node.getExplorationId())]>"
          ng-style="{position: 'absolute', left: '<[$ctrl.pathIconParameters[$index].left]>', top: '<[$ctrl.pathIconParameters[$index].top]>'}">
+
+        <div ng-style="{width: '160px',
```

Comment:
Instead of doing this, go for something like this file: https://github.com/oppia/oppia/blob/37a43ca249ffd2b60bf98f791995048ce0ec5269/core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html. Specifically, move this into a `<style>` tag.

Then handle the left attribute like how we handle it in the library page: https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/library-page/library-page.directive.html